packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts

```diff
@@ -1,5 +1,6 @@
 import { Storage } from 'aws-amplify';
 import { StorageAccessLevel, UploadTask } from '@aws-amplify/storage';
+import { translate } from '@/i18n';
```

Comment:
I believe using `@/<path>` will not work on customer's end -- can we use relative path instead?